frontend/src/scenes/insights/ActionFilter/ActionFilterDropdown.scss

```diff
@@ -1,42 +1,43 @@
 .action-filter-dropdown {
-    .select-box {
-        position: absolute;
-        min-width: 260px;
-        box-shadow: 0 0 8px rgba(0, 0, 0, 0.4);
-        padding: 1rem;
-        border-radius: 2px;
-        border: 1px solid rgba(0, 0, 0, 0.1);
-        height: 400px;
-        background: #fff;
-        z-index: 9999;
-        overflow: visible;
+    position: absolute;
+    min-width: 620px;
+    box-shadow: 0 0 8px rgba(0, 0, 0, 0.4);
+    border-radius: 2px;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    height: 400px;
+    background: #fff;
+    z-index: 999;
+    overflow: visible;
+    .ant-input {
+        border-width: 0 0 1px 0;
+        outline: none;
+    }
+    .ant-divider {
+        margin: 12px 0;
+        border-width: 2px;
     }
-    select {
-        height: 31px;
+
+    .ant-list-item {
+        padding: 4px 12px;
+        cursor: pointer;
+        border: none;
+
+        &:hover,
+        &.selected {
+            background: rgba(0, 0, 0, 0.1);
+        }
     }
-    .select-box-info {
-        position: fixed;
-        background: #fff;
+    .search-list {
+        overflow-y: scroll;
+        height: calc(100% - 32px);
+        padding-top: 4px;
+    }
+    .info-box {
         padding: 1rem;
-        border: 1px solid rgba(0, 0, 0, 0.1);
-        box-shadow: 0 0 8px rgba(0, 0, 0, 0.4);
-        z-index: 99999;
-        opacity: 1;
+        overflow-y: scroll;
+        height: 100%;
         pre {
```

Comment:
This looks broken on my screen:

![Screenshot from 2020-11-19 11-31-55](https://user-images.githubusercontent.com/148820/99647933-19e8a680-2a5b-11eb-86c7-a573f5b41ae4.png)

I think forcing `pre`s to wrap would make sense here.

```white-space: normal``` goes a long way here but still leaves a scroll bar

![Screenshot from 2020-11-19 11-36-00](https://user-images.githubusercontent.com/148820/99648194-6cc25e00-2a5b-11eb-8caf-9820f6b38e5f.png)